nexus/tests/integration_tests/endpoints.rs

```diff
@@ -1601,6 +1585,17 @@ lazy_static! {
             allowed_methods: vec![AllowedMethod::GetNonexistent],
         },
 
+        /* Metrics */
+
+        VerifyEndpoint {
+            url: &DEMO_SYSTEM_METRICS_URL,
+            visibility: Visibility::Public,
+            unprivileged_access: UnprivilegedAccess::None,
+            allowed_methods: vec![
+                AllowedMethod::Get,
+            ],
+        },
```

Comment:
The diff here is confusing, ha. This is just moved down but it wasn't clear at first glance.